### PR TITLE
Add a Pull Request Template

### PR DESCRIPTION
**Pull request by harens**
There are many open PRs that could've been merged if they had followed the [contributing guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md).

As a result, there have been numerous comments such as the following:

> Thanks.
> Can you follow https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md ?

> Thanks. Can you add an entry in the README and the CHANGELOG? More info in https://github.com/lra/mackup/blob/master/CONTRIBUTING.md

Based on the [contributing guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md) and the [following guide](https://github.com/stevemao/github-issue-templates/blob/master/checklist/PULL_REQUEST_TEMPLATE.md), I have written a PR template that should hopefully address this issue.

### CODE REVIEW

**Comment by joshmedeski** [comment]
This is a great idea, thanks for putting it together @harens!

**Review by treee111 — APPROVED** [review]
A PR template is always a good idea.
The checklists-entries you put together are a good start!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* [ ] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
+* [ ] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change
+
+### Adding/updating Application X Support
+
+* [ ] This PR is only for one application
+* [ ] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
+* [ ] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
+
+- [ ] Syncing does not break the application
```
Comment (lra) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: why the newline and the switch between `*` and `-`?

Comment (harens) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: The newline was just to mark a difference between GitHub-specific requirements vs config file requirements, and the change from `*` to `-` allowed for the line break.

However, I'll remove it if that's preferred :)

**Comment by harens** [comment]
Is there anything else required to get this merged? I'd love to help support the project throught this contribution if I can :)
Let me know if I need to squash the commits, although I think that can be down whilst merging.

Thanks again to all the reviewers so far (and for the maintenance of this project).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* [ ] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
+* [ ] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change
+
+### Adding/updating Application X Support
+
+* [ ] This PR is only for one application
+* [ ] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
+* [ ] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
+* [ ] Syncing does not break the application
+* [ ] Syncing does not break any syncing functionality internal to the application
```
Comment (lra) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
* [ ] Syncing does not compete with any syncing functionality internal to the application
```

**Review by lra — COMMENTED** [review]
This is great, my only comment is a clarification about the relationship between Mackup and application that offer a syncing functionality. I always thought Mackup should not compete with those, especially to be fair with the developers who implemented this functionality.

Do you think it's fair?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (harens) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: > This is great, my only comment is a clarification about the relationship between Mackup and application that offer a syncing functionality. I always thought Mackup should not compete with those, especially to be fair with the developers who implemented this functionality.
>
> Do you think it's fair?

That's more than fair enough. Thanks for coming back to this PR @lra, it's greatly appreciated. I'll now try and figure out how to sort out all the commits...

**Review by lra — APPROVED** [review]
Thanks!

**Pull request merged**